config/src/main/java/io/scalecube/config/ConfigRegistryImpl.java

```diff
@@ -73,6 +74,8 @@
   private final Map<String, Map<Class, PropertyCallback>> propertyCallbackMap =
       new ConcurrentHashMap<>();
 
+  private ScheduledFuture<?> schedule;
```

Comment:
1) Why should we store this field? If we store it so maybe we should invoke closing mechanism somewhere, or what is the significance of this field?
2) Can you replace its type to `Disposable`?

Reply:
So I think this field is redundant now.